lib/java/com/google/android/material/bottomappbar/BottomAppBar.java

```diff
@@ -507,6 +507,13 @@ public void performShow() {
     getBehavior().slideUp(this);
   }
 
+  /**
```

Comment:
Maybe also add corresponding isScrollDown()?

Reply:
Sure, I can do that

Reply:
Done!